firestore/integration_test_internal/src/settings_test.cc

```diff
@@ -0,0 +1,74 @@
+// Copyright 2021 Google LLC
+
+#include <future>
+#include <map>
+#include <string>
+
+#include "firebase/firestore.h"
+#include "firestore_integration_test.h"
+
+namespace firebase {
+namespace firestore {
```

Comment:
Nit: please also wrap everything in a unnamed namespace ([link](https://g3doc.corp.google.com/company/readability/home/acquiring-readability/c-readability-advice.md?cl=head#anonymous-namespaces-in-tests)).

Reply:
Done